src/lib/child_processes/child_processes.ml

```diff
@@ -358,20 +346,43 @@ let%test_module _ =
 
     let process_wait_timeout = Time.Span.of_sec 2.1
```

Comment:
I think we could wait less now. But a proper solution is use some kind of signal.